client/src/components/AddQuestion.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div>
+    <slot :clickHandler="show"></slot>
+    <Modal v-show="isShown" @close="close">
+      <template v-slot:header>
+        <Title>Add new question</Title>
+      </template>
+      <QuestionForm
+        v-model="question"
+        :typeOptions="typeOptions"
+        :errors="errors"
+        placeholder="Select question type"
+        @submit="submitQuestion"
+        @close="close"
+      ></QuestionForm>
+    </Modal>
+  </div>
+</template>
+
+<script>
+import styled from 'vue-styled-components';
+import Modal from '@/common/Modal.vue';
+import QuestionForm from '@/components/QuestionForm.vue';
+import { TitleH3 } from '@/common/styledComponents';
+import { toggle } from '@/common/mixins/toggle';
+
+// TODO: remove mock data
+const mockedOptions = [{ id: 1, value: 'Static' }, { id: 2, value: 'Dynamic' }];
```

Comment:
You will remove it in the next pull request, right?

Reply:
I guess it will be removed if we will have these data from BE